configure

```diff
@@ -104,13 +126,17 @@ if (config === 'production') {
     .description(`browserify ${excludes} $in -o $out`)
 }
 
-rule = ninja.rule('browserify-require')
-if (config === 'production') {
-  rule.run(`${browserify} ${requires} $in | ${uglifyjs} > $out`)
-} else {
-  rule.run(`${browserify} ${requires} $in -o $out`)
+function browserifyDependencies() {
+  rule = ninja.rule('browserify-require')
+  if (config === 'production') {
+    rule.run(`${browserify} ${requires} $in | ${uglifyjs} > $out`)
+  } else {
+    rule.run(`${browserify} ${requires} $in -o $out`)
+  }
 }
 
+browserifyDependencies()
+
```

Comment:
Not much point in the function anymore, no?

Reply:
Yeah, that function doesn't make much sense.

Is it still a concern that `rule` is global? 

https://github.com/kitspace/kitspace/pull/266/files/3588916a405a291b8e4eb281b9a68b3232869035#diff-e2d5a00791bce9a01f99bc6fd613a39dR71

https://github.com/kitspace/kitspace/pull/266/files/3588916a405a291b8e4eb281b9a68b3232869035#diff-e2d5a00791bce9a01f99bc6fd613a39dR87

Reply:
Yes and no. I think it's fine to leave it global now and it was my decision in the first place. There might be a better way to do it but I think it's easier on the eyes not coming up with new names for "rule" all the time. 

What made me uneasy about your solution before is that you introduced some non-linearity by using the global "rule" within the function scope. So the rule you were using up top was defined down below without explicitly being passed into the function.

Reply:
Ahh, I see. That makes sense.